docs/ops/cli.md

```diff
@@ -170,29 +170,13 @@ These examples about how to manage a job in CLI.
 
         ./bin/flink cancel <jobID>
 
--   Cancel a job with a savepoint:
+-   Cancel a job with a savepoint (deprecated; use "stop" instead):
 
         ./bin/flink cancel -s [targetDirectory] <jobID>
 
--   Stop a job with a savepoint (streaming jobs only):
+-   Gracefully stop a job (streaming jobs only):
```

Comment:
I would also add "with a savepoint" to make it clear that we always take a savepoint.

Reply:
it might not be a "savepoint" in the future, i.e. it would stop with a checkpoint (or snapshot) or whatnot.

Reply:
Done

Reply:
@aljoscha I went with "savepoint" for now, because that's what it is as of Flink 1.9. "Savepoint" is just in the docs, not in the Public API for the stop command though. 

BTW: another reason not to use `-s` for the `stop` command, although it could also be `snapshot` :)